cumulus_library/studies/core/builder_prereq_tables.py

```diff
@@ -0,0 +1,23 @@
+import pathlib
+
+import sqlparse
+
+from cumulus_library import base_table_builder
+from cumulus_library.helper import get_progress_bar, query_console_output
+
+
+class CorePrereqTableBuilder(base_table_builder.BaseTableBuilder):
+    display_text = "Creating core prerequisite tables..."
+
+    def prepare_queries(self, cursor: object, schema: str, *args, **kwargs):
+        dir_path = pathlib.Path(__file__).resolve().parents[0]
+        prereq_sql = [
+            "version.sql",
```

Comment:
I'm sure this makes sense, but what's the vibe behind moving these files to a prereq table builder?